rfcs/bring-ts.md

```diff
@@ -0,0 +1,248 @@
+# Bring TypeScript
+
+- **Author(s):**: @eladb
+- **Submission Date**: {2023-01-03}
+- **Stage**: Draft
+
+**Goal**: allow developers to use any TypeScript library in their Wing applications.
+
+## Feature Spec
+
+Wing has great interoperability with the TypeScript ecosystem. You can bring any TypeScript library
+and use it Wing.
+
+### ts2wing
+
+In order to use a TypeScript library, we will use the `ts2wing` command to create a Wing Library
+(`.wlib`) from it.
+
+Install `ts2wing` on your system (maybe in the future it will be included in the wing toolchain):
+
+```sh
+$ npm i @winglang/ts2wing
+$ ts2wing --version
+0.1.2
+```
+
+Let's say I want to use the [colors](https://www.npmjs.com/package/colors) library, first, let's
+import it using the `ts2wing` command:
+
+```sh
+$ ts2wing colors
+Installing colors@1.4.0 from npm and all dependencies into vendor/colors.wlib.src...
```

Comment:
option to install via yarn or pnpm in addition to npm would be nice

Reply:
Decoupled the npm installation and the wing bindings, so now users control exactly how to install.